### ISSUE TITLE
* ESP Warns in console *

### ISSUE THREAD

**Issue by Regomy** [Photon-GitHub/AntiCheatAddition#600]
```
[12:05:50 WARN]: [AntiCheatAddition] Task #59 for AntiCheatAddition v9.0.1 generated an exception
com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalStateException: Failed to request texture properties with their UUID a98f0ec7eec2331c928bc2465af40853! Response code: 204
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2203) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at com.google.common.cache.LocalCache.get(LocalCache.java:3937) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3941) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4824) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at de.photon.anticheataddition.util.visibility.PlayerInformationHider.addWatchedPlayerToTablistPacket(PlayerInformationHider.java:113) ~[?:?]
        at de.photon.anticheataddition.util.visibility.legacy.LegacyPlayerHider.onHide(LegacyPlayerHider.java:18) ~[?:?]
        at de.photon.anticheataddition.util.visibility.PlayerInformationHider.setHiddenEntities(PlayerInformationHider.java:101) ~[?:?]
        at de.photon.anticheataddition.util.visibility.PlayerVisibilityImpl.setHidden(PlayerVisibilityImpl.java:20) ~[?:?]
        at de.photon.anticheataddition.modules.additions.esp.Esp.lambda$processWorldQuadTree$8(Esp.java:145) ~[?:?]
        at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:352) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:890) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:412) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:815) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:648) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.lambda$spin$0(MinecraftServer.java:133) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at java.lang.Thread.run(Thread.java:842) [?:?]
Caused by: java.lang.IllegalStateException: Failed to request texture properties with their UUID a98f0ec7eec2331c928bc2465af40853! Response code: 204
        at com.github.retrooper.packetevents.util.MojangAPIUtil.requestPlayerTextureProperties(MojangAPIUtil.java:47) ~[?:?]
        at com.google.common.cache.CacheLoader$FunctionToCacheLoader.load(CacheLoader.java:151) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3527) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2319) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2282) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2197) ~[patched_1.8.8.jar:git-PandaSpigot-"858ab0f"]
```

**Comment by Janmm14** [comment]
not related to esp

**Comment by Photon-GitHub** [comment]
Should be fixed in 9.1.0
